repos/system_upgrade/el7toel8/actors/cupsfilterscheck/actor.py

```diff
@@ -0,0 +1,49 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import Report, InstalledRedHatSignedRPM, CupsFiltersModel
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+from subprocess import Popen, PIPE
+
+class CupsfiltersCheck(Actor):
+    """
+    cupsfilters_check actor
+
+    Checks if cups-filters package is installed and if cups-browsed.conf was
+    modified by user, which prevents automatic replacement by package installer.
+    """
+
+    name = 'cupsfilters_check'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report, CupsFiltersModel)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+      installed_cupsfilters = False
+      migrateable = False
```

Comment:
If I understood correctly both variables, installed_cupsfilters and migrateable, have always the same values. Either both True or both False. I think we could simplify this by only using one of them.